TBRM Tests scripts consistency with  te2 fixes

marchemi opened the pull request:
According : [Fixed TBRM Test Plan issues reported during TE2](https://github.com/CHIP-Specifications/chip-test-plans/pull/4529) : 
**Correct**  
- Test_TC_TBRM_2_2.yaml
- Test_TC_TBRM_2_3.yaml

**Adding**

- Test_TC_TBRM_2_4.yaml (Thank to @vivien-apple  for needed chip-tool modification to introduce PASE session)
- Test_TC_TBRM_3_1_Simulated.yaml

StephaneGUELEC committed:
Add files via upload

Add yaml test script for TBRM

StephaneGUELEC committed:
Merge branch 'master' into master

StephaneGUELEC committed:
Update TEST_TC_TBRM_2.2.yaml

StephaneGUELEC committed:
Update TEST_TC_TBRM_2.3.yaml

StephaneGUELEC committed:
Update TEST_TC_TBRM_2.4.yaml

StephaneGUELEC committed:
Merge branch 'master' into master

marchemi committed:
Merge branch 'project-chip:master' into master

marchemi committed:
Merge branch 'project-chip:master' into master

marchemi committed:
Test script consitancy wit test plan after TE2

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/project-chip/connectedhomeip?pullRequest=35153) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/project-chip/connectedhomeip?pullRequest=35153) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: StephaneGUELEC<br/>:x: marchemi<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/project-chip/connectedhomeip?pullRequest=35153) it.</sub>

marchemi committed:
Test script consitancy wit test plan after TE2

marchemi committed:
Merge branch 'master' into tbrm-te2-fixes

marchemi committed:
Merge branch 'master' into tbrm-te2-fixes

marchemi committed:
Merge branch 'project-chip:master' into tbrm-te2-fixes

marchemi committed:
Update src/app/tests/suites/certification/Test_TC_TBRM_2_3.yaml

Co-authored-by: Karsten Sperling <113487422+ksperling-apple@users.noreply.github.com>

marchemi committed:
Update src/app/tests/suites/certification/Test_TC_TBRM_2_2.yaml

Co-authored-by: Karsten Sperling <113487422+ksperling-apple@users.noreply.github.com>

restyled-commits committed:
Restyled by whitespace

restyled-commits committed:
Restyled by prettier-yaml

marchemi committed:
Merge branch 'master' into tbrm-te2-fixes

marchemi committed:
Merge branch 'master' into tbrm-te2-fixes

marchemi committed:
Test_TC_TBRM_2_4. synchronisation with TC-THNETDIR-2.3 according test Plan

restyled-commits committed:
Restyled by whitespace

marchemi committed:
Merge branch 'master' into tbrm-te2-fixes

ksperling-apple committed:
Test tweaks to get CI to pass

- Use pairing payload matching the other parameters
- Check response of ArmFailSafe commands
- Fix bad merge in commissioner_commands.py

marchemi committed:
Merge pull request #227 from ksperling-apple/tbrm-te2-fixes

Test tweaks to get CI to pass

restyled-commits committed:
Restyled by prettier-yaml
